### PR TITLE
Setting Up a to do list

### PR DESCRIPTION
**Pull request by Caren-Koroeny**
- Set up to do list  based on the webpack.
- Create an empty To Do List placeholder `div` or `ul` element .
- Create an index.js file and set an array of some simple to do tasks (array of objects);
- Each task object should contain three keys:
                - `description [string]`
                -  `completed [bool]`
                -  `index: [number]`
 - Write a function to iterate over the tasks array and populate an HTML list item element for each task.
 - render the dynamically created list of tasks in the dedicated placeholder.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  justify-content: space-between;
+  align-items: center;
+  align-content: center;
+}
+
+.add-new-task-container {
+  display: flex;
+  flex-direction: row;
+  gap: 170px;
+}
```
Comment (ShadyShawkat) [inline src/style.css]: - [x] Please make sure that the add button doesn't stick out of the design to match the original template design.

image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Caren-Koroeny, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (Caren-Koroeny) [reply src/style.css]: @ShadyShawkat  thank you, I corrected it.
Screenshot from 2023-01-23 17-34-35

**Review by gemmen29 — APPROVED** [review]
Hi @Caren-Koroeny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**